library/alloc/src/vec.rs

```diff
@@ -287,6 +287,7 @@ use crate::raw_vec::RawVec;
 /// [`insert`]: Vec::insert
 /// [`reserve`]: Vec::reserve
 /// [owned slice]: Box
+/// [`&`]: ../../std/primitive.reference.html
```

Comment:
Shouldn't this link to the slice primitive, not references?

Reply:
Yes, but the text specifically says 'To get a slice, use `&`.' So either 'slice' should link to slices or `&` should, but only putting a link to references seems confusing.

Reply:
Okay, I do think `slice` should link to slice and `&` should link to reference.